### ISSUE TITLE
WSA apps NOT scaling/spanning across multiple monitors.

### ISSUE THREAD

**Issue by AgrMayank** [microsoft/WSA#199]
### Steps to reproduce

- Open any WSA app
- Try to resize it across your monitors
- Alternatively use apps like Asus ScreenPad plus which basically resizes the app across monitors.

### ✔️ Expected Behavior

The app should properly scale/span/resize across the monitors with the content being displayed across both of them just like what happens with any other native app on windows.

### ❌ Actual Behavior

What is happening is, the app either doesn't scale more than the monitor resolution and even if it can (using Asus Screenpad Plus), it scales and shows a blank panel on the secondary monitor. This used to work earlier, but after a few updates, it no longer works. 

### Other Software

The bug happens with any android app installed on WSA. I use Apple Music and Tachiyomi primarily installed using APK. I also use Asus Screenpad Plus which basically resizes the app to fill/span across multiple monitors.

### Please specify the version of Windows Subsystem for Android

2210.40000.7.0

**Comment by hamza-usmani** [comment]
Hi, could you provide screenshots/video and details about your screen resolution? We'd like to understand exactly what's happening here.

**Comment by AgrMayank** [comment]
https://user-images.githubusercontent.com/26793209/210750760-b4c44713-cc9c-46af-8917-95134a27ea9c.mp4

If you check the file above you'll know what I mean. The content on the 2nd display isn't being rendered at all. This didn't happen in earlier builds!

**Comment by nieubank** [comment]
Thanks for feedback here, we're aware of this issue internally as well. Unfortunately, this is an Android limitation, Android itself doesn't support apps spanning multiple display devices. Addressing this issue is not on the immediate roadmap but we'll continue to evaluate going forward.

**Comment by AgrMayank** [comment]
Hmm, not sure then why it was working in the earliest builds of WSA, but it's a much-needed feature as it opens whole new possibilities. Would be really looking forward to it.

**Comment by nieubank** [comment]
> Hmm, not sure then why it was working in the earliest builds of WSA, but it's a much-needed feature as it opens whole new possibilities. Would be really looking forward to it.

In the earliest of WSA builds from 2021 this did "work" due to WSA using a virtual display model to represent windows. These could be of arbitrary size but with that came numerous issues compromising compatibility, maintainability with upstream Android and performance, this the switch to the current free form windowing model. 

We have in the past done a proof of concept on a fox but usage data shows that this scenario is extremely rare so it hasn't been prioritized. 

If you could elaborate on the new possibilities you have in mind we will consider that in regards to prioritization.

**Comment by AgrMayank** [comment]
> In the earliest of WSA builds from 2021 this did "work" due to WSA using a virtual display model to represent windows. These could be of arbitrary size but with that came numerous issues compromising compatibility, maintainability with upstream Android and performance, this the switch to the current free form windowing model.

Thanks for the clarification! I guess that's why the maximum window size is dependent on the monitor size it is running on.

> We have in the past done a proof of concept on a fox but usage data shows that this scenario is extremely rare so it hasn't been prioritized.
> 
> If you could elaborate on the new possibilities you have in mind we will consider that in regards to prioritization.

So, the MAJOR use case would be as an e-book/comic book reader in cases where you're using a stacked monitors layout or even dual-screen laptops like Zenbook Duo/Yoga Book 9i, which are becoming increasingly common as of late. Due to this, I was able to use my laptop as a Kindle of sorts for reading which was really nice and my main use case of WSA!

You can also take advantage of it while gaming on 2 side by side monitors, or just fill your multi-monitor setup for more display area thus greater immersion. People do these with windows apps every time. 

Think of Surface Duo with 2 screens side by side. It does open a lot of possibilities.

**Comment by alpha54net** [comment]
> If you could elaborate on the new possibilities you have in mind we will consider that in regards to prioritization.

Now that laptops/devices with two screens are becoming more coming, this feature would be awesome. I use a Lenovo Yoga Book 9i for example, and if I want to read a Kindle book or magazine across both screens, I need to use the Kindle Cloud Reader in my browser as I can stretch the browser across both screens. It would be so much more useful to be able to span native Android apps like Kindle and other magazine readers across the two screens.

Though I fear with WSA being deprecated (which is so disappointing - this is the top feature that has allowed me to travel with a single Windows touchscreen device - I'm not excited to have to start bringing a separate tablet again), I imagine any changes on this front are rather unlikely, hey :(

**Comment by AgrMayank** [comment]
> WSA being deprecated (which is so disappointing - this is the top feature

Exactly, this is so unfortunate. WSA has been one of the top 3 features of Windows for me in a long long time!!! It was so awesome and didn't have any of the headaches of other emulators.

I'm seriously hoping they can open-source it (99% sure they won't).
